sdk/graphrbac/graph/README.md

```diff
@@ -1,6 +1,14 @@
 ## Azure GraphRbacManagementClient SDK for JavaScript
 
-This package contains an isomorphic SDK for GraphRbacManagementClient.
+This package contains an isomorphic SDK for GraphRbacManagementClient to work with the Azure Active Directory Graph API which is now deprecated. Starting June 30th 2022, we will end support for Azure AD Graph and will no longer provide technical support or security updates. Apps using Azure AD Graph after this time will no longer receive responses from the Azure Active Directory Graph endpoint.
+
+We recommend you move to using the newer [Microsoft Graph API](https://developer.microsoft.com/graph) instead.
```

Comment:
Nit: I think it'd be nice to put this notice above the description as a forewarning. If it's below the description, it might get swallowed by someone skimming the page:

"**Warning**: The Azure Active Directory Graph API is deprecated and has been replaced by the Microsoft Graph API. Beginning on **June 30th, 2022**, apps using Azure AD Graph after this time **will no longer receive responses** from the Azure Active Directory Graph endpoint."

Then the normal looking description stuff and migration guide after.